src/components/Staking/Nominator/Nominated/styles.module.scss

```diff
@@ -1,3 +1,4 @@
 .checkbox {
     zoom: 1.5;
+    margin-left: 17px;
```

Comment:
is that really the best way to center it lol?

Reply:
Yes, the cell itself is too big and it looks out of place since table headings aren't centered (that would look uglier)